packages/core/config.js

```diff
@@ -88,9 +88,14 @@ module.exports.schema = {
       return includes(BREADCRUMB_TYPES, maybeType)
     }, true))
   },
+  context: {
+    defaultValue: () => undefined,
+    message: 'should be a string',
+    validate: value => value === undefined || typeof value === 'string'
+  },
   user: {
     defaultValue: () => null,
-    message: '(object) user should be an object',
+    message: 'user should be an object',
```

Comment:
Could remove 'user' here? I guess the context would be sufficient to know it was user? Also makes it consistent with the other messages and may save a byte if we're lucky!

Reply:
6 bytes after minification!